mk/riscv-arch-test.mk

```diff
@@ -1,3 +1,9 @@
+riscof-check:
+	$(Q)if [ "$(shell pip show riscof 2>&1 | head -n 1 | cut -d' ' -f1)" = "WARNING:" ]; then \
+	$(PRINTF) "Run 'pip3 install git+https://github.com/riscv/riscof.git@d38859f85fe407bcacddd2efcd355ada4683aee4' to install RISCOF\n"; \
```

Comment:
Is this workaround still making sense?

Reply:
> Is this workaround still making sense?

This may not be the solution for issue #506, but if someone attempts to run `make arch-test` instead of executing the script ".ci/riscv-tests.sh", the target will exit because riscof is not the default Python module. This alert can easily clarify any confusion.

Reply:
Done.